config/default.yml

```diff
@@ -2931,6 +2931,11 @@ Style/CollectionMethods:
     detect: 'find'
     find_all: 'select'
     member?: 'include?'
+  # Methods in this array accept a final symbol as an implicit block
+  # eg. `inject(:+)`
+  MethodsAcceptingSymbol:
+    - inject
+    - reduce
```

Comment:
I implemented the methods that accept a symbol instead of a block as a configuration so that it can be extended if necessary. I am not aware of any other methods that have this ability but please let me know if I've missed something.